content/docs/develop/reference/cli/index.md

```diff
@@ -234,26 +237,57 @@ stages:
 To create a new project, use the command `create project` and specify the name of the project as well as the `shipyard.yaml` file.
 
 ```console
-$ keptn create project sockshop shipyard.yml
+keptn create project your_project shipyard.yml
 ```
 
 ## keptn onboard service
 
 After creating a project which represents a repository in your GitHub organization, the keptn&nbsp;CLI allows to onboard services into this project. Please note that for describing the Kubernetes resources, [Helm charts](https://helm.sh/) are used. Therefore, the keptn CLI allows setting a Helm values description in the before created project. Optionally, the user can also provide a Helm deployment and service description.
 
-To onboard a service, use the command `onboard service` and provide the project name, the Helm chart values and optionally also deployment and service descriptions.
+To onboard a service, use the command `onboard service` and provide the project name (flag `--project`), the Helm chart values (flag `--values`) and optionally also deployment (flag `--deployment`) and service (flag `--service`) descriptions.
 
 ```console
-$ keptn onboard service --project=sockshop --values=values.yaml
+keptn onboard service --project=your_project --values=values.yaml
 ```
 or
 ```console
-$ keptn onboard service --project=sockshop --values=values.yaml --deployment=deployment.yaml --service=service.yaml
+keptn onboard service --project=your_project --values=values.yaml --deployment=deployment.yaml --service=service.yaml
 ```
 
 To start onboarding a service, please see the [Onboarding a Service](../../usecases/onboard-carts-service) use case.
 
+## keptn send event new-artifact
+
+After onboarding a service, the keptn&nbsp;CLI allows pushing a new artifact for the service.
+This artifact is a Docker image, which can be located at Docker Hub, Quay, or any other registry storing docker images.
+The new artifact is pushed in the first stage specified in the  `shipyard.yaml` file (usually this will be the dev-stage).
+Afterwards, keptn takes care of deploying this new artifact.
+
+To push a new artifact, use the command `send event new-artifact`, which sends a new-artifact-event 
+to keptn in order to deploy a new artifact for the specified service in the provided project.
+Therefore, this command takes the project (flag `--project`), the service (flag `--service`), 
+as well as the image (flag `--image`) and tag (flag `--tag`) of the new artifact.
+
+```console
+keptn send event new-artifact --project=your_project --service=your_service --image=docker.io/keptnexamples/carts --tag=0.7.0
```

Comment:
Didn't we decide to have the option to say `--image=docker.io/keptnexamples/carts:0.7.0` ?